README.md

```diff
@@ -81,6 +81,9 @@ async function showPerson(person) {
 }
 ```
 
+Prefixed names can be written with a colon between quotes (`person['foaf:knows']`) or by using an underscore (`person.foaf_knows`) or dollar sign (`person.foaf$knows`).
+When the name contains an underscore, such as `foaf:based_near`, then remove the underscore with the context (`"basedNear": "based_near"`).
```

Comment:
So `foaf:based_near` does not work? Sounds to me like it should.

But `based_near` itself indeed wouldn't work, that would need to be `basedNear` as in your explanation.

So just double-checking here that `foaf:based_near` indeed fails; if so, that's likely a bug.

Reply:
person['foaf:based_near'] does not work when only  "@vocab": "http://xmlns.com/foaf/0.1/" is defined in the context. It needs "foaf": "http://xmlns.com/foaf/0.1/" too

Reply:
Ok. But in that case, `foaf:whatever` also wouldn't work, so it's not specific to the underscore.

Could you change the `such as foaf:based_near` phrasing slightly to convey this? Thanks.